.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+cache: pip
+sudo: required
+services:
+  - docker
+
+# Upgrade to latest docker-engine
+before_install:
+  - sudo apt-get update -qq
+  - sudo DEBIAN_FRONTEND=noninteractive apt-get install -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" docker-engine
```

Comment:
docker is already installed, isn't it? it works out of the box https://github.com/ome/omero-mapr/blob/master/.travis.yml ?